hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/regression/factories/AtomicBatchProviderFactory.java

```diff
@@ -0,0 +1,202 @@
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.files.RandomFile;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.files.RandomFileDeletion;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.files.RandomFileInfo;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.files.RandomFileUpdate;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.meta.RandomReceipt;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.meta.RandomRecord;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.schedule.RandomSchedule;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.schedule.RandomScheduleDeletion;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.schedule.RandomScheduleInfo;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.schedule.RandomScheduleSign;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.token.RandomToken;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.token.RandomTokenAccountWipe;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.token.RandomTokenAssociation;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.token.RandomTokenBurn;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.token.RandomTokenDeletion;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.token.RandomTokenDissociation;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.token.RandomTokenFreeze;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.token.RandomTokenInfo;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.token.RandomTokenKycGrant;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.token.RandomTokenKycRevoke;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.token.RandomTokenMint;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.token.RandomTokenTransfer;
+import com.hedera.services.bdd.spec.infrastructure.providers.ops.token.RandomTokenUnfreeze;
+import com.hedera.services.bdd.spec.infrastructure.selectors.RandomSelector;
+import com.hedera.services.bdd.spec.props.JutilPropertySource;
+import com.hederahashgraph.api.proto.java.AccountID;
+import com.hederahashgraph.api.proto.java.ContractID;
+import com.hederahashgraph.api.proto.java.FileID;
+import com.hederahashgraph.api.proto.java.Key;
+import com.hederahashgraph.api.proto.java.ResponseCodeEnum;
+import com.hederahashgraph.api.proto.java.ScheduleID;
+import com.hederahashgraph.api.proto.java.TokenID;
+import com.hederahashgraph.api.proto.java.TopicID;
+import java.io.File;
+import java.util.function.Function;
+
+public class AtomicBatchProviderFactory {
+
+    public static final String RESOURCE_DIR = "eet-config";
+
+    static HapiPropertySource propsFrom(final String resource) {
+        final var loc = RESOURCE_DIR + File.separator + resource;
```

Comment:
`Paths.get(RESOURCE_DIR).resolve(resource)` would likely provide a more robust implementation, but not a big deal